### ISSUE TITLE
entity_edit task crashes if file is missing from IA

### ISSUE THREAD

**Issue by gjost** [denshoproject/ddr-local#299]
User tried to save `ddr-densho-435-1-1` and got the following error.  According to the logic of `DDR.archivedotorg.is_iaobject` and `DDR.archivedotorg.get_ia_metadata` there should be metadata at the Internet Archive but there is not yet.
```
Traceback (most recent call last):
  File "/opt/ddr-local/venv/ddrlocal/lib/python3.7/site-packages/celery/app/trace.py", line 412, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/opt/ddr-local/venv/ddrlocal/lib/python3.7/site-packages/celery/app/trace.py", line 704, in __protected_call__
    return self.run(*args, **kwargs)
  File "/opt/ddr-local/ddrlocal/webui/tasks/entity.py", line 82, in entity_edit
    git_name, git_mail, collection, form_data
  File "/opt/ddr-local/ddrlocal/webui/models.py", line 773, in save
    docstore.Docstore().post(self)
  File "/opt/ddr-local/venv/ddrlocal/lib/python3.7/site-packages/ddr_cmdln-5.2.3-py3.7.egg/DDR/docstore.py", line 578, in post
    public_fields=public_fields, public=public, b2=b2
  File "/opt/ddr-local/venv/ddrlocal/lib/python3.7/site-packages/ddr_cmdln-5.2.3-py3.7.egg/DDR/models/common.py", line 420, in to_esobject
    d.ia_meta = archivedotorg.get_ia_meta(self)
  File "/opt/ddr-local/venv/ddrlocal/lib/python3.7/site-packages/ddr_cmdln-5.2.3-py3.7.egg/DDR/archivedotorg.py", line 29, in get_ia_meta
    raise FileNotFoundError(f'No Internet Archive data for {o.identifier.id}.')
FileNotFoundError: No Internet Archive data for ddr-densho-435-1-1.
```
Task should catch the error and not cause a crash.

**Comment by gjost** [comment]
Fixed in `ddr-local` commit `b0c67bd` for package `ddrlocal-master-5.2.4`.

**Comment by gjost** [comment]
For good measure, just in case some state has crept into my test VM, I spun up a completely new and pristine Debian 10 VM and installed the `ddrlocal-master_5.2.4~deb10_amd64.deb` package on it.  After getting `ddr` set up with Gitolite credentials I `git clone`-ed `ddr-densho-435` and edited `ddr-densho-435-1-1` with success.  Again `ddrindex` ran with the same results as in the last comment.

I don't think this is a problem with the code.  It's some problem with the setup on the VM, or with the collection itself.  Maybe that machine is sitting near a source of magnetism or cosmic rays?

**Comment by gjost** [comment]
We need to catch FileNotFoundError in webui/tasks/files.py too.
Sara encountered error with ddr-densho-435-1-1:
```
Traceback (most recent call last):
  File "/opt/ddr-local/venv/ddrlocal/lib/python3.7/site-packages/celery/app/trace.py", line 412, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/opt/ddr-local/venv/ddrlocal/lib/python3.7/site-packages/celery/app/trace.py", line 704, in __protected_call__
    return self.run(*args, **kwargs)
  File "/opt/ddr-local/ddrlocal/webui/tasks/files.py", line 483, in set_signature              < < < < < < < < < < < < < < < <
    {}
  File "/opt/ddr-local/ddrlocal/webui/models.py", line 773, in save
    docstore.Docstore().post(self)
  File "/opt/ddr-local/venv/ddrlocal/lib/python3.7/site-packages/ddr_cmdln-5.2.3-py3.7.egg/DDR/docstore.py", line 578, in post
    public_fields=public_fields, public=public, b2=b2
  File "/opt/ddr-local/venv/ddrlocal/lib/python3.7/site-packages/ddr_cmdln-5.2.3-py3.7.egg/DDR/models/common.py", line 420, in to_esobject
    d.ia_meta = archivedotorg.get_ia_meta(self)
  File "/opt/ddr-local/venv/ddrlocal/lib/python3.7/site-packages/ddr_cmdln-5.2.3-py3.7.egg/DDR/archivedotorg.py", line 29, in get_ia_meta
    raise FileNotFoundError(f'No Internet Archive data for {o.identifier.id}.')
FileNotFoundError: No Internet Archive data for ddr-densho-435-1-1.
```

**Comment by gjost** [comment]
Fixed in`ddr-local` commit `24d53b9` in package `ddrlocal-master_5.2.6`.
